index.rst

```diff
@@ -383,6 +210,7 @@ week without reprocessing all alerts available since then.
 Daily topics also make expiring nights of data straightforward
 instead of ending up expiring data somewhere in the middle
 of the night.
+However, daily topics require more manual management by downstream consumers, and large numbers of Kafka topics can create stability issues. 
```

Comment:
Yeah, I might want to come back and argue against daily topics more thoroughly... I should do some writing on it to clarify my own thoughts.

Reply:
Right, I thought we would want to discuss it.  I'll add a weasel sentence that we're still thinking about it.